Dockerfile

```diff
@@ -7,15 +7,16 @@ USER       root
 RUN        yum install -y epel-release
 RUN        yum repolist
 RUN        yum install -y git python-pip python-devel
+RUN	   yum install -y gcc openssl-devel
 RUN        pip install --upgrade pip
 RUN        useradd -d /home/flasker -m flasker
```

Comment:
Not worth fixing at this point.  I need to take all the microservices and make them happy under uwsgi, not running standalone.  When I have a Docker recipe that does that well, then we can make uwsgi run as something sane.